tutorials/epochs/plot_10_epochs_overview.py

```diff
@@ -173,10 +173,10 @@
 # Basic visualization of ``Epochs`` objects
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 #
-# The :class:`~mne.Epochs` object can be visualized (and browsed interactively)
-# using its :meth:`~mne.Epochs.plot` method:
+# The :class:`~mne.Epochs` object can be visualized with its events (and
+# browsed interactively) using its :meth:`~mne.Epochs.plot` method:
 
-epochs.plot(n_epochs=10)
+epochs.plot(n_epochs=10, events=epochs.events)
```

Comment:
I'm actually leaning slightly toward *not* showing events here...  we do demo that in the next tutorial (https://mne.tools/dev/auto_tutorials/epochs/plot_20_visualize_epochs.html#plotting-epochs-as-time-series), maybe it's good to show here what it looks like with the defaults?

Also, the paragraph that comes right after this needs to be updated (even if you kept in the events, it still mentions "green line", and if you remove events as I'm suggesting, then we shouldn't say "the event ID is marked on the top axis" :)

Reply:
Good point, maybe it would be nice to have `plot_20_visualize_epochs` show up as an example of `Epochs.plot` in the doc? I think the `['face']` key is preventing this however.